config.md

```diff
@@ -28,7 +28,8 @@ Changing it means creating a new derived image, instead of changing the existing
 A layer DiffID is a SHA256 digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., `sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9`.
 Layers must be packed and unpacked reproducibly to avoid changing the layer DiffID, for example by using tar-split to save the tar headers.
 
-NOTE: the DiffID is different than the digest in the manifest list because the manifest digest is taken over the gzipped layer for `application/vnd.oci.image.layer.v1.tar+gzip` types.
+The difference between DiffIDs and the layer digests in the [manifest's `layers`](manifest.md#image-manifest-property-descriptions) is that the layer digest is taken over the blob regardless of compression, while the DiffID is taken after removing any compression.
+For an `application/vnd.oci.image.layer.tar+gzip` layer, the layer digest is taken over the `application/vnd.oci.image.layer.tar+gzip` content, while the DiffID is take over the `application/vnd.oci.image.layer.tar` content.
```

Comment:
> This may not always be a given.

What may not be given?  Are you just talking about the last sentence?  It's qualified with “For an `application/vnd.oci.image.layer.tar+gzip` layer” (although that's missing the `v1` added in #347), and I think the rest of that sentence is always true in that case.  Of course, there may be layers that are not `application/vnd.oci.image.layer.v1.tar+gzip`…

> A layer digest may be taken over the uncompressed content.  The only limitation is that DiffID is always uncompressed

Isn't that what I'm saying (e.g. with “regardless of compresson” for layers and “after removing any compression” for DiffIDs)?  Which wording in particular do you think is saying something different, and what do you think it's saying.  Once we identify the sticky phrasing we can reword it for additional clarity.